### PR TITLE
Fix graphical bug with match puzzle, default block height is now "40".

### PR DESCRIPTION
**Pull request by DrewPerlman**
left: before, right: after match update before after

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -1446,10 +1446,28 @@ a.download-video {
   #answerdest {
     width: 240px;
   }
+  .answerlist {
+    h4 {
+      position: relative;
+      top: 25%;
+      -webkit-transform: translateY(-50%);
+      -ms-transform: translateY(-50%);
+      transform: translateY(-50%);
```
Comment (breville) [inline dashboard/app/assets/stylesheets/application.css.scss]: i suspect scss has a mixin that will generate the browser-specific variations, so you don't have to manually type them all.

**Comment by breville** [comment]
also, this fix sounds cool.. any chance you could post a before & after screenshot to see what's fixed?

**Comment by laurelfan** [comment]
maybe you should add a test using the eyes screenshot diff tool that brian
just set up?

**Comment by bcjordan** [comment]
@laurelfan good idea, I had added one for another match puzzle but it looks like it doesn't have the issue that gets fixed here, I'll add one covering the level in that screenshot. & I'll link to the PR here.

Kind of a pain to get the key & login set up at the moment, but should have that good and documented by end of week!

**Comment by DrewPerlman** [comment]
left: before, right: after
Text lined up:
multi text lined up
Picture multi's slightly affected:
multi picture affected

**Comment by breville** [comment]
The improved images look great.

**Comment by DrewPerlman** [comment]
This PR is finished and ready for merging!

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -1366,6 +1365,12 @@ a.download-video {
   }
 }
 
+@mixin yTranslation($offset) {
```
Comment (breville) [inline dashboard/app/assets/stylesheets/application.css.scss]: interesting, i didn't realise we had to create our own mixin for this.  scss doesn't already have support?

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 3]
```diff
@@ -1446,10 +1449,24 @@ a.download-video {
...
+      top: 25%;
+      @include yTranslation(-50%);
+    }
+  }
   #answers {
     list-style-type: none;
     margin-left: 0px;
   }
+  .answerslot {
+    h4 {
```
Comment (breville) [inline dashboard/app/assets/stylesheets/application.css.scss]: if we're taking h4 out of the answers, then why use it in the css?

File: dashboard/app/assets/stylesheets/application.css.scss (hunk 2 above)
Comment (DrewPerlman) [inline dashboard/app/assets/stylesheets/application.css.scss]: Was not aware of any pre-created mixins.

File: dashboard/app/assets/stylesheets/application.css.scss (hunk 3 above)
Comment (DrewPerlman) [inline dashboard/app/assets/stylesheets/application.css.scss]: My mistake, this was a change I had made before I knew we were taking out the h4's from the multi puzzles. This change still works though, as it only affects one of the match puzzles that use h4. And it looks better, in my opinion. The match puzzle I'm referring to is featured in the picture I uploaded in the description.

**Comment by breville** [comment]
I tested the following levels on chrome and ie8 and they seem to work fine:

http://localhost.studio.code.org:3000/s/course3/stage/10/puzzle/3
http://localhost.studio.code.org:3000/s/course3/stage/10/puzzle/2
http://localhost.studio.code.org:3000//s/course3/stage/9/puzzle/2
http://localhost.studio.code.org:3000/s/course3/stage/19/puzzle/2
http://localhost.studio.code.org:3000/s/course1/stage/11/puzzle/12
http://localhost.studio.code.org:3000/s/course2/stage/7/puzzle/14

**Pull request merged**